src/pages/MinerDashboard/Payments/Payments.chart.tsx

```diff
@@ -53,10 +54,7 @@ const PaymentsChart: React.FC<{ address: string; coin?: ApiPoolCoin }> = ({
       paymentsAxis.renderer.grid.template.disabled = true;
       let dateAxis = paymentsChart.xAxes.push(new DateAxis());
       dateAxis.renderer.grid.template.location = 0;
-      dateAxis.baseInterval = {
```

Comment:
As quoted [here](https://www.amcharts.com/docs/v4/concepts/axes/date-axis/#Manually_specifying_data_granularity), `baseInterval` should only be used when you _know_ the time granularity of the data source.